src/main/java/core/basesyntax/service/file/FileWriterServiceImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.file;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterServiceImpl implements FileWriterService {
+    @Override
+    public void write(String data, String path) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(path))) {
+            writer.write(data);
+        } catch (IOException e) {
+            throw new RuntimeException("Cannot write to file: " + path);
```

Comment:
Consider including the original `IOException` as a cause in the `RuntimeException` to provide more context for debugging. For example: `throw new RuntimeException("Cannot write to file: " + path, e);`